misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/jdi/StackFrameProxyImpl.java

```diff
@@ -302,7 +302,7 @@ public Value getValue(LocalVariableProxyImpl localVariable) throws JdiProxyExcep
         throw new JdiProxyException(error.getMessage(), error);
     }
 
-    private Map<LocalVariable, Value> getAllValues() throws JdiProxyException {
+    public Map<LocalVariable, Value> getAllValues() throws JdiProxyException {
```

Comment:
Lets make this private

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/31279/commits/8b8ba6a04de50d8d9a98f0622372775b3c2b9262